### PR TITLE
[SNDVOL32] Small Italian translation update

### PR DESCRIPTION
**Pull request by danielecampione**
[SNDVOL32] Better Italian translation.

## Purpose
Better Italian translation.

## Proposed changes
I want to enhance the Italian translation.

### CODE REVIEW

**Comment by HBelusca** [comment]
Cc @Fraizeraust , @ElCresh .

File: base/applications/sndvol32/lang/it-IT.rc [hunk 1]
```diff
@@ -113,24 +113,24 @@ END
 
 IDD_ADVANCED DIALOGEX 0, 0, 255, 140
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Advanced Controls for Volume Control"
+CAPTION "Controlli avanzati per il controllo del volume"
```
Comment (ElCresh) [inline base/applications/sndvol32/lang/it-IT.rc]: Maybe better without repetition like "Controlli avanzati per il volume"

**Comment by danielecampione** [comment]
Ok, I'll fix it and I'll create a new Pull request.

**Comment by learn-more** [comment]
@danielecampione You do not need to create a new PR for a small fix.

**Pull request closed without merging**